src/action_types/apps.ts

```diff
@@ -2,6 +2,9 @@
 // See LICENSE.txt for license information.
 import keyMirror from 'utils/key_mirror';
 
+// This file's contents belong to the Apps Framework feature.
```

Comment:
Are these changes part of this effort: https://mattermost.atlassian.net/browse/MM-32447 ?

Reply:
Yes, updated the description.